lib/index.js

```diff
@@ -4,10 +4,12 @@ export default function init({
   videoUrl,
   audioUrl = '',
   options,
-  proxyUrl = 'http://webrtc-proxy-001.rnfrst.com:8000'
+  proxyUrl = 'http://webrtc-proxy-001.rnfrst.com:8000',
+  onError = (err) => console.log(err),
 }) {
   let webrtcServer = new WebRtcStreamer(elemId, proxyUrl);
   webrtcServer.connect(videoUrl, audioUrl, undefined);
+  webrtcServer.onError = onError;
```

Comment:
@AJFunk you set the `onError` after calling `connect`. the problem is that you use `onError` in connect so it would just be undefined. 

I would recommend passing it in the constructor here https://github.com/rainforestapp/webrtc-proxy-client/pull/2/files#diff-6d186b954a58d5bb740f73d84fe39073R10

and then setting `this.onError = onError` to make sure its available from the beginning of the class instantiation

Reply:
oh yeah. duh 👍

I suppose that's what happens when we don't have specs... we should do that soon